client/src/helpers/APICalls/dropDates.ts

```diff
@@ -0,0 +1,18 @@
+import { AuthApiData } from '../../interface/AuthApiData';
+import { FetchOptions } from '../../interface/FetchOptions';
+
+const dropDates = async (where: string, dropIn: Date, dropOut: Date): Promise<AuthApiData> => {
+  const fetchOptions: FetchOptions = {
+    method: 'GET',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ where, dropIn, dropOut }),
+    credentials: 'include',
+  };
+  return await fetch(`/getInfo`, fetchOptions)
```

Comment:
```suggestion
  return await fetch(`/get-info`, fetchOptions)
```
We use hyphenated case for route names in JS :)

Reply:
Done the changes.